tests/foreman/cli/test_contenthost.py

```diff
@@ -53,34 +52,38 @@ def setUp(self):  # noqa
                 {u'organization-id': TestContentHost.NEW_ORG['id']},
                 cached=True)
         if TestContentHost.LIBRARY is None:
```

Comment:
Do we want to standardize the capitalization on our test cases class constants? I am fine with the upper case, but I think most of the other CLI tests use lower case. Maybe this could be something we tackle after GA :)